ha/core/system_health/health_evaluators/element_health_evaluator.py

```diff
@@ -81,10 +82,10 @@ def get_children(self, element: str, element_id: str, **kwargs) -> dict:
         if len(children) == 0:
             return {}
         key = ElementHealthEvaluator.prepare_key(element, comp_id=element_id, **kwargs)
-        key = key.replace("/health", "").replace("/", "", 1)
+        key = key.replace(HA_DELIM+"health", "").replace(HA_DELIM, "", 1)
```

Comment:
Please check if you can use f"" string here.

Reply:
as discussed changed everywhere